pom.xml

```diff
@@ -539,7 +539,7 @@
       <plugin>
         <artifactId>maven-failsafe-plugin</artifactId>
         <configuration>
-          <argLine>-XX:MaxMetaspaceSize=128m ${jacoco.agent.it.arg}</argLine>
+          <argLine>-XX:MaxMetaspaceSize=512m ${jacoco.agent.it.arg}</argLine>
```

Comment:
The last time I wanted to up the memory for `failsafe`, @fasseg was of the vociferous opinion that it wold be difficult for people to build Fedora with a half-gig for tests. I disagreed, but we left it alone then. You might want to ping the `-tech` list letting them know this is happening, in case anyone wants to react.